roles/bigboot/files/bigboot.sh

```diff
@@ -249,12 +249,10 @@ convert_size_to_fs_blocks() {
 }
 
 deactivate_volume_group() {
-    local volume_group_name
-    volume_group_name=$(get_volume_group_name)
-    ret=$(/usr/sbin/lvm vgchange -an "$volume_group_name" 2>&1)
+    ret=$(/usr/sbin/lvm vgchange -an "$LVM2_VG_NAME" 2>&1)
```

Comment:
You're not using `get_volume_group_name`, is it used somewhere else? If no, please remove it. If yes, why use two methods for the same information?

Reply:
Right, `get_volume_group_name` in not used anywhere and I removed it. All of the LVM info is now populated with this line in `check_available_free_space`...
```
eval "$(/usr/sbin/lvm pvs --noheadings --nameprefixes --nosuffix --units b -o vg_name,vg_extent_size,pv_pe_count,pv_pe_alloc_count,vg_free_count "$device")"
```